eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/TxBody.hs

```diff
@@ -104,63 +104,63 @@ import NoThunks.Class (NoThunks (..))
 
 -- =======================================================
 
-data TxBodyRaw era = TxBodyRaw
-  { inputs :: !(Set (TxIn (EraCrypto era))),
-    outputs :: !(StrictSeq (ShelleyTxOut era)),
-    certs :: !(StrictSeq (DCert (EraCrypto era))),
-    wdrls :: !(Wdrl (EraCrypto era)),
-    txfee :: !Coin,
-    vldt :: !ValidityInterval, -- imported from Timelocks
-    update :: !(StrictMaybe (Update era)),
-    adHash :: !(StrictMaybe (AuxiliaryDataHash (EraCrypto era))),
-    mint :: !(MultiAsset (EraCrypto era))
+data ShelleyMATxBodyRaw era = ShelleyMATxBodyRaw
```

Comment:
For consistency it should be just `MATxBodyRaw`  (in order to match `MATxBody`), which also means that prefix  will need to be switched from `smatbr` -> `matbr`
```suggestion
data MATxBodyRaw era = MATxBodyRaw
```

"Shelley" in the naming of the package is an artifact, so prefixing types with `Shelley` doesn't make much sense (especially since we are planning on splitting into two separate packages `Mary` and `Allegra`

Reply:
Totally Agree!